common-tools/clas-detector/src/main/java/org/jlab/detector/decode/DetectorEventDecoder.java

```diff
@@ -76,31 +76,37 @@ public final void initDecoderDev(){
         tablesFitter = Arrays.asList(new String[]{"/daq/fadc/clasdev/htcc"});
         translationManager.init(keysTrans,tablesTrans);
         fitterManager.init(keysFitter, tablesFitter);
-        scalerManager.init(Arrays.asList(new String[]{"/runcontrol/fcup","/runcontrol/slm","/runcontrol/hwp","/runcontrol/helicity","/daq/config/scalers/dsc1"}));
+        scalerManager.init(Arrays.asList(new String[]{"/runcontrol/fcup","/runcontrol/slm","/runcontrol/hwp",
+                                                      "/runcontrol/helicity","/daq/config/scalers/dsc1"}));
     }
 
     public final void initDecoder(){
-        keysTrans = Arrays.asList(new String[]{
-		"FTCAL","FTHODO","FTTRK","LTCC","ECAL","FTOF","HTCC","DC","CTOF","CND","BST","RF","BMT","FMT","RICH","HEL","BAND","RTPC","RASTER"
-        });
 
+        // Detector translation table
+        keysTrans = Arrays.asList(new String[]{"FTCAL","FTHODO","FTTRK","LTCC","ECAL","FTOF",
+                                               "HTCC","DC","CTOF","CND","BST","RF","BMT","FMT",
+                                               "RICH","HEL","BAND","RTPC","RASTER","ATOF"
+        });
         tablesTrans = Arrays.asList(new String[]{
             "/daq/tt/ftcal","/daq/tt/fthodo","/daq/tt/fttrk","/daq/tt/ltcc",
             "/daq/tt/ec","/daq/tt/ftof","/daq/tt/htcc","/daq/tt/dc","/daq/tt/ctof","/daq/tt/cnd","/daq/tt/svt",
             "/daq/tt/rf","/daq/tt/bmt","/daq/tt/fmt","/daq/tt/rich2","/daq/tt/hel","/daq/tt/band","/daq/tt/rtpc",
-            "/daq/tt/raster"
+            "/daq/tt/raster","/daq/tt/atof2"
```

Comment:
So we should delete both /daq/tt/atof and /daq/tt/atof2, and then you recreate the former and use it here.

Reply:
That sounds good. I have a simple script to recreate it.

Reply:
@baltzell 
ccdb create table:
```
ccdb -c CONNECTION  mktbl atof -r 720 crate=int slot=int channel=int sector=int layer=int component=int order=int
```
Download https://userweb.jlab.org/~whit/data_files/atof_tt_init.txt
fill table:
```
ccdb -c CONNECTION add /daq/tt/atof atof_tt_init.txt
```

Reply:
What about these:

 create         whit             2024-10-22 21-56-41    Created directory '/test/alert'
 create         whit             2024-10-22 22-01-31    Created table with path '/test/alert/crate=int'
 create         whit             2024-10-22 22-03-07    Created table with path '/test/alert/atof'

Reply:
done:  https://clasweb.jlab.org/cgi-bin/ccdb/show_request?request=/daq/tt/atof:0:default:2024-10-29_20-44-28